src/store/chat/saga.ts

```diff
@@ -70,10 +73,45 @@ export function* setActiveConversation(id: string) {
   history.push({ pathname: `/conversation/${id}` });
 }
 
+function* validateActiveConversation() {
+  const [conversationList, activeConversationId] = yield all([
+    select(rawConversationsList()),
+    select(activeConversationIdSelector),
+  ]);
+
+  if (!activeConversationId || conversationList.length === 0) {
+    return;
+  }
+
+  const isMemberOfActiveConversation = conversationList.includes(activeConversationId);
+  yield put(setIsConversationErrorDialogOpen(!isMemberOfActiveConversation));
+}
+
+function* watchForConversationChange() {
```

Comment:
I guess the case when the user isn't a part of the conversation, and he still tries to access it would be when he has an unknown url, and tries to load it, in which case we can just move this check to `setActiveConversationId`? (i.e validate the conversation before setting the active conversationId ?)